### PR TITLE
XW-4866 | Show full main page content

### PR DESCRIPTION
**Pull request by JamesHawking**
## Links

* https://wikia-inc.atlassian.net/browse/XW-4866

### CODE REVIEW

File: app/components/full-main-page-content.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import Component from '@ember/component';
+import {computed} from '@ember/object';
+
+export default Component.extend({
+	classNames: ['full-main-page-content'],
+	actions: {
+		openSection() {
+			this.set('openSection', !this.get('openSection'));
```
Comment (latata) [inline app/components/full-main-page-content.js]: `this.toggleProperty('openSection')`
leniuszek nie sprawdził!

I'd define default value (`false`) `openSection` property in the component

File: app/templates/components/full-main-page-content.hbs [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<h2 section="1" {{action 'openSection'}} class="{{if openSection 'open-section'}}">
+	<div class="section-header-label">
+		{{i18n 'app.view-more'}}
```
Comment (latata) [inline app/templates/components/full-main-page-content.hbs:3]: I think `view more` label doesn't fit perfectly here.

File: app/templates/components/full-main-page-content.hbs [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+	</div>
+	{{svg 'wds-icons-menu-control-tiny' class='wds-icon wds-icon-small chevron'}}
+</h2>
+<section class="article-content mobile-hidden">
+	{{article-content
+			content=fullMainPageContent
```
Comment (latata) [inline app/templates/components/full-main-page-content.hbs]: 1 tab less please!

File: app/templates/components/main-page.hbs [hunk 4]
```diff
@@ -24,4 +24,8 @@
 	{{#if trendingVideos}}
 		{{trending-videos model=trendingVideos}}
 	{{/if}}
+
+	{{#if fullMainPageContent}}
+		{{full-main-page-content fullMainPageContent=fullMainPageContent}}
```
Comment (latata) [inline app/templates/components/main-page.hbs]: I'd go with `{{full-main-page-content content=fullMainPageContent}}`.

Comment (JamesHawking) [reply app/templates/components/main-page.hbs]: 👍

File: app/templates/components/full-main-page-content.hbs (hunk 2 above)
Comment (JamesHawking) [reply app/templates/components/full-main-page-content.hbs:3]: Let's talk on daily about it.

File: app/templates/components/full-main-page-content.hbs [hunk 5]
```diff
@@ -0,0 +1,11 @@
+<h2 section="1" {{action 'openSection'}} class="{{if openSection 'open-section'}}">
+	<div class="section-header-label">
+		{{i18n 'app.view-more'}}
+	</div>
+	{{svg 'wds-icons-menu-control-tiny' class='wds-icon wds-icon-small chevron'}}
```
Comment (rybmat) [inline app/templates/components/full-main-page-content.hbs]: `wds-menu-control-tiny` and `wds-icon-small` class?

**Review by rybmat — CHANGES_REQUESTED** [review]

File: app/templates/components/full-main-page-content.hbs [hunk 6]
```diff
@@ -0,0 +1,11 @@
+<h2 section="1" {{action 'openSection'}} class="{{if openSection 'open-section'}}">
```
Comment (hakubo) [inline app/templates/components/full-main-page-content.hbs:1]: do we have to do some shenanigans like here https://github.com/Wikia/mobile-wiki/blob/dev/app/components/article-content.js#L516
when opening this section?

Comment (JamesHawking) [reply app/templates/components/full-main-page-content.hbs:1]: You mean aria-shenanigans? If not, it's as simple as it is.

File: app/templates/components/full-main-page-content.hbs (hunk 2 above)
Comment (JamesHawking) [reply app/templates/components/full-main-page-content.hbs:3]: View more approved by Stanley.

File: app/templates/components/full-main-page-content.hbs (hunk 6 above)
Comment (hakubo) [reply app/templates/components/full-main-page-content.hbs:1]: nope. gibberish.
article-content is actually rendered below a whole... 
don't listen to me ;)

**Review by hakubo — APPROVED** [review]

**Review by rybmat — APPROVED** [review]

**Pull request merged**